Android/app/src/main/java/io/github/project_travel_mate/MainActivity.java

```diff
@@ -37,7 +37,9 @@
 import org.json.JSONObject;
 
 import java.io.IOException;
+import java.util.ArrayList;
```

Comment:
why add these imports? These are not being used

Reply:
I had some debugging functions inside main activity. 

I tested the library to see if country flags populated correctly. Forgot to optimize the imports after i removed the code. Now fixed.